feature-libs/product-configurator/rulebased/styles/_configurator-show-more.scss

```diff
@@ -2,22 +2,16 @@
   &:not(:empty) {
     font-size: 14px;
     line-height: 1.2;
-
-    @include forFeature('productConfiguratorAttributeTypesV2') {
-      padding-block-end: 10px;
-    }
+    padding-block-end: 10px;
 
     button {
       background-color: transparent;
       border: none;
       display: inline-block;
-      outline-offset: 0px;
+      outline-offset: 0;
```

Comment:
Why that change, has the previous state been a typo? Because that's not related to the toggle?